TSC-Charter.md

```diff
@@ -2,37 +2,30 @@
 
 ## Section 1. Guiding Principle.
 
-The Node.js Foundation will operate transparently, openly,
-collaboratively, and ethically. Project proposals, timelines, and status
-must not merely be open, but also easily visible to outsiders.
+The Node.js project is part of the OpenJS Foundation which
+operates transparently, openly, collaboratively, and ethically.
+Project proposals, timelines, and status must not merely be
+open, but also easily visible to outsiders.
 
-## Section 2. Evolution of Node.js Foundation Governance.
+## Section 2. Evolution of OpenJS Foundation Governance.
 
 Most large, complex open source communities have both a business and a
-technical governance model. Node.js Foundation’s technical leadership
-is the Technical Steering Committee (“TSC”). Node.js Foundation’s business
+technical governance model. Technical leadership for the projects
+within the OpenJS Foundation are delegated to the projects through
+their project charter by the OpenJS Cross Project Council (CPC).
+In the case of the Node.js project it is delegated to the Node.js
+Technical Steering Committee (“TSC”). OpenJS Foundation’s business
 leadership is the Board of Directors (the “Board”).
 
 This Technical Steering Committee Charter reflects a carefully
-constructed balanced role for the TSC and the Board in the governance of
-Node.js Foundation. The charter amendment process is for the TSC to
+constructed balanced role for the TSC and the CPC in the governance of
+the OpenJS Foundation. The charter amendment process is for the TSC to
 propose changes using simple majority of the full TSC, the proposed
-changes being subject to review and approval by the Board. The Board may
+changes being subject to review and approval by the CPC. The CPC may
 additionally make amendments to the TSC charter at any time, though the
-Board will not interfere with day-to-day discussions, votes or meetings
+CPC will not interfere with day-to-day discussions, votes or meetings
 of the TSC.
 
-## Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.  
-
-The Board will set the overall TSC Policy. The policy will describe the
-overarching scope of the Node.js Foundation initiative, Node.js
-Foundation’s technical vision and direction and project release
-expectations in the form of expected cadence and intent. The Board will
-use the TSC as a delegate body for governing technical implementation,
-individual project scope and direction while they remain within the scope
-and direction of the policies as described in the TSC Policy document and
-approved by the Board.
-
 ## Section 4. Establishment of the TSC.
```

Comment:
Section numbers should be updated on the whole doc

Reply:
@mcollina thanks for catching that.